sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationAsyncClient.java

```diff
@@ -892,25 +781,397 @@ public Mono<List<DictionaryLookupItem>> lookupDictionaryEntries(String from, Str
+     * provide
+     * client trace id.
+     * </p>
+     *
+     * @param texts Defines the content of the request.
+     * @param clientTraceId A client-generated GUID to uniquely identify the request.
+     * @param language Language tag identifying the language of the input text.
+     * If a code isn't specified, automatic language detection will be applied.
+     * @param script Script tag identifying the script used by the input text.
+     * If a script isn't specified, the default script of the language will be assumed.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<BreakSentenceItem>> findSentenceBoundaries(List<String> texts, String clientTraceId,
+        String language, String script) {
+        return findSentenceBoundariesInner(convertTextToData(texts), clientTraceId, language, script);
+    }
+
+    /**
+     * Find Sentence Boundaries.
+     * <p>
+     * This method is used when you have single text for which you want to find sentence boundaries and you want to
+     * provide
+     * client trace id.
+     * </p>
+     *
+     * @param text Defines the content of the request.
+     * @param clientTraceId A client-generated GUID to uniquely identify the request.
+     * @param language Language tag identifying the language of the input text.
+     * If a code isn't specified, automatic language detection will be applied.
+     * @param script Script tag identifying the script used by the input text.
+     * If a script isn't specified, the default script of the language will be assumed.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<List<BreakSentenceItem>> findSentenceBoundaries(String text, String clientTraceId, String language,
```

Comment:
Do we want `clientTraceId` on some of these overloads? If this is an advanced scenario, we should just keep the clientTraceId only on the `WithResponse` methods.

Reply:
Done